WeaponPack/src/main/java/com/github/sef24sp4/weaponpack/shotgun/ShotGun.java

```diff
@@ -42,11 +74,13 @@ private List<ShotGunBullet> createBullets(final IEntity shooter){
 	}
 
 	private double getRandomRotation(final IEntity shooter){
-		double leftDeviation = shooter.getRotation() - 1;
-		double rightDeviation = shooter.getRotation() + 1;
+		double minimumRotation = shooter.getRotation() - 1;
+		double maximumRotation = shooter.getRotation() + 1;
+		double changedRotation = random.nextDouble(minimumRotation,maximumRotation);
```

Comment:
This part also requires feedback, since a value is defined for -1 and +1 for left and right projectiles. Should the value of 1 be replaced with maxDeviationFromRotation?? I wrote +1 and -1 on my notes from the code-session??